narwhal/primary/src/proposer.rs

```diff
@@ -374,15 +377,6 @@ impl Proposer {
         }
 
         let mut enough_votes = votes_for_leader >= self.committee.validity_threshold();
-        if enough_votes {
-            if let Some(leader) = self.last_leader.as_ref() {
-                debug!(
-                    "Got enough support for leader {} at round {}",
```

Comment:
ooc why dont we want this log anymore?

Reply:
This is printed on every call to the function after there are enough votes, so moved to a different callsite and printed only on the transition.